src/pages/upgrades/index.js

```diff
@@ -382,6 +382,33 @@ const Eth2IndexPage = ({ data }) => {
                 <Translation id="page-upgrades-what-happened-to-eth2-5" />
               </Link>
             </p>
+            <H2>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-title" />
+            </H2>
+            <h3>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-title" />
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-description" />
+            </p>
+            <h3>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-title" />
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-description" />
+            </p>
+            <h3>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-title" />
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-description" />
+            </p>
+            <h3>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-title" />
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-description" />
+            </p>
```

Comment:
I think the hierarchy here works better bringing these headings down a level.

<img width="1338" alt="Screenshot 2022-01-27 at 15 12 00" src="https://user-images.githubusercontent.com/62268199/151387172-91b23fc1-6a7a-4807-849a-ed454e7ad41d.png">

```suggestion
            <h3>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-title" />
            </h3>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-description" />
            </p>
```